core/CMake_readelf.cmake

```diff
@@ -129,23 +129,31 @@ if (check_libc)
   endif (readelf_result OR readelf_error)
 
   if (BUILD_PACKAGE)
```

Comment:
This CMake script is launched separately and so variables in the configure process, like BUILD_PACKAGE, will not be set here: they would have to be passed in explicitly on the command line.  But, we already have a check_libc parameter, so I think we just update the callers who are now passing ON to instead pass the value of BUILD_PACKAGE.